libs/api/domains/national-registry/src/lib/models/familyMember.model.ts

```diff
@@ -10,11 +13,13 @@ export class NationalRegistryFamilyMember {
   @Field(() => String)
   fullName!: string
 
-  @Field(() => String)
-  gender!: string
+  @IsEnum(Gender)
```

Comment:
This is combining dtos and GraphQL types.
Don't need the class-validator on graphql objects.